### ISSUE TITLE
Check for update prior to updating

### ISSUE THREAD

**Issue by cliff1wright** [andDevW/getChromium#10]
Great app.  I use it a lot.

One suggestion - have the app check to see if there is an update available so I don't have to DL the whole file just to find out if there is an update.

THX,

**Comment by andDevW** [comment]
I can add in an option setting to "enable update notifications".

**Comment by Primokorn** [comment]
Good point and having the version number would be perfect. Thanks!

**Comment by andDevW** [comment]
After looking into it further, I don't know if this would be a useful feature. Chromium has two build types, Last Known Good Revision(LKGR) and Latest. LKGR was the default for getChromium v1.1, as it is the most stable/usable build(not really very buggy) and it is updated very infrequently. The current build hasn't been updated in [over 2 months](https://download-chromium.appspot.com/?platform=Android&type=continuous). The Latest build is more buggy, but is constantly being updated. The most recent Latest build was updated (at the time of writing)[around 18 minutes ago](https://download-chromium.appspot.com/?platform=Android&type=snapshots) and will be updated again very soon. I've added the Latest build to getChromium v2.1, so now it's possible to install either build type interchangeably.

The problem with a notification is that it would either be all the time (for Latest) or a few times a year (for LKGR). Maybe adding back the version numbers would be a better way of addressing the issue?
